MaxMind.Db/Reader.cs

```diff
@@ -247,7 +252,10 @@ private int StartNode(int bitLength)
         private T Deserialize<T>(JToken value)
         {
             var serializer = new JsonSerializer();
-            return serializer.Deserialize<T>(new JTokenReader(value));
+            using (var reader = new JTokenReader(value))
+            {
+                return serializer.Deserialize<T>(reader);
+            }
         }
```

Comment:
[The Dispose() method](https://github.com/ayoung/Newtonsoft.Json/blob/master/Newtonsoft.Json/JsonReader.cs#L449) seems to just change the state on the token reader. I don't think it could cause an actual resource leak, but it is still good practice to dispose of it properly.